calblueprint/siren
# calendar screens

# ✨ New in this PR ✨

## One liner: what did you do? 🚨 
front-end, styling, and switch toggling for calendar schedule page
globally, created a new ScrollContainer (as opposed to PageContainer) which doesn't center vertically and allows for vertical scrolling

## Coverage 🙆‍♀️
_use this section to break up your task into submodules and track progress_ 
 - [X] upcoming screen 
 - [X] schedule new screen
 - [X] toggle/switch component & functionality

## How can the reviewer test your code? 👩‍💻
_example: run `npm start` and click through Navigation bar_ 
go to schedule page (liphoebe01@gmail.com pw: password) and check how it looks
login with some other account or login without wifi to see edge cases (when no appointments / upcoming)

## Any bugs you encountered or still having trouble with? 🐛
_describe how you solved them such that someone can follow your explanation_
- have to add add <></> to end of upcoming body because otherwise it'll bleed onto next screen. not sure why this happens
- appointments take case type as camelCase, not sure if i should always convert camelCase to Titlecase or if we could modify database
- appointments (in upcoming) have a date format im not sure how to handle/format - any possibility i can be given this date as a Date and not a string?
- there's a bit of bottom padding on the upcoming screen, not sure where this is coming from, not super noticeable though

## Resources 📔
_link online resources and related PRs_
https://github.com/calblueprint/siren/pull/32

🧜‍♀️ cc: @shannonbonet @phoebeli23
